core/src/main/java/org/eclipse/hono/config/ProtocolAdapterProperties.java

```diff
@@ -49,4 +50,47 @@ public final boolean isAuthenticationRequired() {
     public final void setAuthenticationRequired(final boolean authenticationRequired) {
         this.authenticationRequired = authenticationRequired;
     }
+
+    /**
+     * Checks if adapter should include <em>Vendor Properties</em> as defined by <a
+     * href="http://docs.oasis-open.org/amqp-bindmap/jms/v1.0/wd09/amqp-bindmap-jms-v1.0-wd09.pdf">
```

Comment:
For me, this URL does not work - but this one: https://www.oasis-open.org/committees/download.php/60574/amqp-bindmap-jms-v1.0-wd09.pdf

Reply:
Thanks for spotting. :+1:  I took the URL from the document itself but it seems that it will be the URL where the document can be found once it has been released (i.e. not a working draft anymore). I will update the links ...